### ISSUE TITLE
App crashes if all required apps are installed

### ISSUE THREAD

**Issue by SanchithHegde** [jahirfiquitiva/Kuper#27]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests.
- [x] I have verified that I am using the latest version of Kuper.

### Device/dashboard info:
 - Kuper version: `1.3.2`
 - Android version: `8.1`
 - Device Manufacturer: `Motorola`
 - Device Name: `Moto G3 2015`

### Reproduction Steps
1. Install all required apps on device.
2. Open app.

### Expected Result
App doesn't crash on opening.

### Actual Result
App crashes on opening.

### Logcat
```
05-17 12:54:50.729 W/Glide (14494): Failed to find GeneratedAppGlideModule. You should include an annotationProcessor compile dependency on com.github.bumptech.glide:compiler in your application and a @GlideModule annotated AppGlideModule implementation or LibraryGlideModules will be silently ignored
05-17 12:54:50.796 W/AHBottomNavigation(14494): The items list should have at least 3 items 
05-17 12:54:50.808 W/AHBottomNavigation(14494): The items list should have at least 3 items 
05-17 12:54:51.064 E/AndroidRuntime(14494): FATAL EXCEPTION: main 
05-17 12:54:51.064 E/AndroidRuntime(14494): Process: jahirfiquitiva.apps.kuper.sample, PID: 14494 
05-17 12:54:51.064 E/AndroidRuntime(14494): java.lang.NullPointerException: You cannot start a load on a not yet attached View or a Fragment where getActivity() returns null (which usually occurs when getActivity() is called before the Fragment is attached or after the Fragment is destroyed). 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at com.bumptech.glide.util.Preconditions.checkNotNull(Unknown Source:4) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at com.bumptech.glide.Glide.getRetriever(Unknown Source:2) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at com.bumptech.glide.Glide.with(Unknown Source:4) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.fragments.KuperFragment$kuperAdapter$2.invoke(Unknown Source:17) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.fragments.KuperFragment$kuperAdapter$2.invoke(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at kotlin.SynchronizedLazyImpl.getValue(Unknown Source:24) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.fragments.KuperFragment.getKuperAdapter(Unknown Source:2) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.fragments.KuperFragment.setupWallpaperInAdapter(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.fragments.KuperFragment.setUserVisibleHint(Unknown Source:20) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.support.v4.app.FragmentStatePagerAdapter.setPrimaryItem(Unknown Source:27) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.support.v4.view.ViewPager.populate(Unknown Source:469) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.support.v4.view.ViewPager.populate(Unknown Source:2) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.support.v4.view.ViewPager.setAdapter(Unknown Source:134) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.widgets.PseudoViewPager.setAdapter(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.activities.KuperActivity.initPagerAdapter(Unknown Source:120) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.activities.KuperActivity.setupBottomNavigation(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.activities.KuperActivity.access$setupBottomNavigation(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.activities.KuperActivity$setupContent$1.invoke(Unknown Source:2) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at jahirfiquitiva.libs.kuper.ui.activities.KuperActivity$setupContent$1.invoke(Unknown Source:0) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at ca.allanwang.kau.utils.UtilsKt$sam$Runnable$7a0dd3de.run(Unknown Source:2) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.os.Handler.handleCallback(Handler.java:790) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.os.Handler.dispatchMessage(Handler.java:99) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.os.Looper.loop(Looper.java:164) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at android.app.ActivityThread.main(ActivityThread.java:6499) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at java.lang.reflect.Method.invoke(Native Method) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:442) 
05-17 12:54:51.064 E/AndroidRuntime(14494): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:807) 
05-17 12:54:51.069 W/ActivityManager(1241): Force finishing activity jahirfiquitiva.apps.kuper.sample/.MainActivity 
05-17 12:54:51.081 W/ContextImpl(1241): Calling a method in the system process without a qualified user: android.app.ContextImpl.sendBroadcast:966 com.android.server.am.AppErrors.crashApplicationInner:393 com.android.server.am.AppErrors.crashApplication:321 com.android.server.am.ActivityManagerService.handleApplicationCrashInner:14524 com.android.server.am.ActivityManagerService.handleApplicationCrash:14506 
05-17 12:54:51.083 W/BroadcastQueue(1241): Permission Denial: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to ProcessRecord{476b35c 12446:eu.thedarken.sdm/u0a194} (pid=12446, uid=10194) requires android.permission.READ_LOGS due to sender android (uid 1000) 
05-17 12:54:51.087 E/AppCrashReceiver(12476): jahirfiquitiva.apps.kuper.sample stopped unexpectedly...
```

I've tested this on a device running Android 7.x and experienced the same issue.

**Comment by jahirfiquitiva** [comment]
@SanchithHegde 

Can you please test with version code `dcae214` and let me know if the issue is fixed?

**Comment by jahirfiquitiva** [comment]
Should be fixed in v `1.3.3`
